### DISCUSSION TITLE
特斯拉廣告誇大車子續航力，遭南韓重罰 220 萬美元

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#558] [General]
韓聯社、[路透社]報導，南韓公平交易委員會（Korea Fair Trade Commission，KFTC）3日宣布，依法對特斯拉核處28.5億韓圜罰緩。KFTC表示，自2019年8月以來，特斯拉一直在當地刊登的廣告上宣稱，旗下車型每充電一次「至少可跑」數百公里，但事實上是誇大其辭。
KFTC指出，特斯拉在廣告上宣稱的續航里程，只有在氣溫為平均值且行駛在市中心才可能達成，如果是在極寒天氣，特斯拉電動車的實際續航里程短少了50.5%之多
https://technews.tw/2023/01/03/south-korea-fines-tesla-2-2-mln-for-exaggerating-driving-range-of-evs/

**Comment by HsuChen88** [comment]
首先要先有錢可以買特斯拉，希望有朝一日我也能充電一次跑百里。

**Comment by skyling0299** [comment]
這是很嚴重的問題，因為如果謊報車子續航力導致人判斷錯誤的話有廣告不實的傾向。同時有趣的是電力在雪地裡會降低，電動車是否有其限制?

**Comment by 0212Jerry** [comment]
我看過有人比較ioniq5 和特斯拉的比較，特斯拉在電動車上的地位基本上很難撼動，光是車上的系統可以不停的更新就已經收買了大批的消費者。撇除標題的問題外，我自己是覺得特斯拉整體還是非常優的。

**Comment by Vincent20030314** [comment]
廣告不實很不應該，尤其是這種大公司，雖然我也沒錢買特斯拉就是了。

**Comment by yjpxi** [comment]
對於誇大宣傳的企業進行罰款，能促進市場上的公平競爭，確保所有企業在相同的規則下運營，鼓勵企業以產品實際性能為基礎進行競爭，而不是依賴誇大其詞的廣告。
